### PR TITLE
Fallback: Run zmq build script inside gyp file

### PR DESCRIPTION
**Pull request by lgeiger**
This is a bit hacky but it will allow to rebuild `zmq-prebuilt`.

This allows us to use `zmq-prebuilt` with newer electron releases.

### CODE REVIEW

**Comment by lgeiger** [comment]
Closes #42

**Comment by rgbkrk** [comment]
Game on!

File: .travis.yml [hunk 1]
```diff
@@ -33,9 +32,8 @@ before_install:
  - npm --version
  - test "$(uname)" = "Darwin"  || export CXX=g++-4.9 CC=gcc-4.9
  - gcc --version
- - sh build_libzmq.sh
 
-install: env PKG_CONFIG_PATH=$ZMQ_PREFIX/lib/pkgconfig npm install
+install: npm install
```
Comment (rgbkrk) [inline .travis.yml]: Woot

File: README.md [hunk 2]
```diff
@@ -45,39 +45,30 @@ Now, prepare to be amazed by the wonders of binaries.
 
 ## Usage
 
-Replace `require(zmq)` in your code base with `zmq-prebuilt`. That's it.
+Replace `require(zmq)` in your code base with `require(zmq-prebuilt)`. That's it.
```
Comment (rgbkrk) [inline README.md]: :+1:

File: README.md [hunk 3]
```diff
@@ -45,39 +45,30 @@ Now, prepare to be amazed by the wonders of binaries.
...
 The wonder of binaries begins.
 
 ----
 
 ## Installation - Contributors and Development
 
 To set up `zmq-prebuilt` for development, fork this repository and
-clone your fork to your system. Be sure you have `git-lfs` installed.
+clone your fork to your system. Be sure you have Python 2 and `git-lfs` installed.
```
Comment (rgbkrk) [inline README.md]: le sad day

File: README.md [hunk 4]
```diff
@@ -45,39 +45,30 @@ Now, prepare to be amazed by the wonders of binaries.
...
-Install a development version of `zmq-prebuilt` with the following:
-
-```bash
-./build_libzmq.sh
-npm install
-```
+**Prerequisites for Windows**
 
-### Windows
+On Windows you'll need [Visual Studio 2013](https://www.microsoft.com/en-US/download/details.aspx?id=44914).
```
Comment (rgbkrk) [inline README.md]: We won't be foiled on AppVeyor here?

File: binding.gyp [hunk 5]
```diff
@@ -26,6 +26,7 @@
           ],
         }],
         ['OS=="mac" or OS=="solaris"', {
+          'install_zmq': '<!(./build_libzmq.sh 2>&1 > zmq-build.log)',
```
Comment (rgbkrk) [inline binding.gyp]: Awesome

File: README.md (hunk 4 above)
Comment (lgeiger) [inline README.md]: Appveyor has almost every version installed. [This line](https://github.com/nteract/zmq-prebuilt/blob/master/binding.gyp#L11) makes sure that gyp uses  Visual Studio 2013.

I tried it on a system where only Visual Studio 2015 was installed and gyp complained.

Comment (rgbkrk) [inline README.md]: Awesome.

**Pull request merged**